Foundations/LisBella.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LisBella.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 28/03/23.
+//
+
+import SwiftUI
+
+struct LisBella: View {
+    var body: some View {
+        ProfileCard(
+            title: "Conexões"
+        ) {
+            HStack(spacing: -10) {
+                ProfileCircle("profile",
+                              radius: 45)
+                    .frame(height: 25)
+                ProfileCircle("kiev-gama",
+                              radius: 45)
+                    .frame(height: 25)
+                Text("...").font(.custom(
+                        "Inter-Light",
+                        size: 16
+                )).padding(.leading, 15)
+            }
+        }
+    }
+}
```

Comment:
The struct name is not clarifying and it is not documented... Can you provide some details about it?

Reply:
This struct is a test file that will be removed before the merge. I explained in the PR description. The name is the same of the teacher's app